src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderMap.cs

```diff
@@ -47,59 +75,152 @@ private CoseHeaderMap(bool isReadOnly)
             }
         }
 
+        /// <summary>
+        /// Gets the CBOR encoded value associated with the specified label, as a signed integer.
+        /// </summary>
+        /// <param name="label">The label of the value to get.</param>
+        /// <returns>The decoded value associated with the specified label, as a signed integer.</returns>
+        /// <exception cref="InvalidOperationException">Decodification failed, see the inner exception for the exact exception that was thrown.
+        /// -or
```

Comment:
Applies to other members in this file as well.

```suggestion
        /// -or-
```

It's not clear to me though if `<para>` tags are required for the new line contents.

Reply:
> It's not clear to me though if <para> tags are required for the new line contents.

It is not clear to me either. My approach was to go with the most popular way.
Using VS Code search with files to include: `src\**\*.cs`
`/// -or-` yields 279 results in 42 files.
`/// <para>-or-` yields 170 results in 25 files.


cc @carlossanlop

Reply:
Be careful in counting, since some of the ones that don't look like they used `<para>` just used more newlines, e.g.

https://github.com/dotnet/runtime/blob/6ee429c704e52c48cee7902f76df7b358f8c19ec/src/libraries/System.ComponentModel.Composition/src/System/ComponentModel/Composition/Hosting/AssemblyCatalog.cs#L59-L61

Reply:
My understanding is that they are required.  Otherwise, XML normalization would say that they're the same as if you just wrote them at the end of the line.  To get the newlines to matter, you need the `<para>`